### PR TITLE
I found this... and I had to.

### PR DESCRIPTION
**Pull request by KazeEspada**
This was about 4-5 years ago, and it's just there!

### CODE REVIEW

File: code/modules/food&drinks/drinks/drinks.dm [hunk 1]
```diff
@@ -110,7 +110,7 @@
 	spillable = 1
 
 /obj/item/weapon/reagent_containers/food/drinks/golden_cup/tournament_26_06_2011
-	desc = "A golden cup. It will be presented to a winner of tournament 26 june and name of the winner will be graved on it."
+	desc = "A golden cup. It will be presented to a winner of tournament 26 june and name of the winner will be graved on it. For some reason, it's still not engraved."
```
Comment (KorPhaeron) [inline code/modules/food&drinks/drinks/drinks.dm]: Should be engraved anyway

Comment (KazeEspada) [inline code/modules/food&drinks/drinks/drinks.dm]: Who's name?

Comment (KorPhaeron) [inline code/modules/food&drinks/drinks/drinks.dm]: No I mean change "graved" to "engraved"

Or maybe just let it have a generic description so admins can hand it out whenever.

Comment (KazeEspada) [inline code/modules/food&drinks/drinks/drinks.dm]: The generic one is above this one?

Comment (KorPhaeron) [inline code/modules/food&drinks/drinks/drinks.dm]: Nevermind me then

**Comment by PKPenguin321** [comment]
google "to vs too vs two"

File: code/modules/food&drinks/drinks/drinks.dm (hunk 1 above)
Comment (optimumtact) [inline code/modules/food&drinks/drinks/drinks.dm]: what if the winner's name was actually graved?

**Comment by octareenroon91** [comment]
>2555

>Name still not graved

**Comment by Cheridan** [comment]
instead of adding a silly comment you could
get rid of it

**Comment by optimumtact** [comment]
>adds paper memes
Calls others out on memes

**Comment by Cheridan** [comment]
you're a meme

**Comment by optimumtact** [comment]
At least I own it

**Pull request closed without merging**